src/code.c

```diff
@@ -119,26 +119,31 @@ static inline void PopLoopNesting( void ) {
   STATE(LoopNesting) = STATE(LoopStack)[--STATE(LoopStackCount)];
 }
 
-static inline void SetupGapname(TypInputFile* i)
```

Comment:
Did you benchmark this? SetupGapname gets called every time we call NewStat, and you are now doing an ELM_LIST on every call.

Reply:
If you want to go down rewriting this code, you could switch back to returning void, and just move the ELM_LIST into a different tiny function, for the places where we want it (that function could call SetupGapname to ensure the values have all been installed).

Reply:
I didn't realize it was also being called by NewStat. I'll change it.